### PR TITLE
Ifhaam's classes

### PR DESCRIPTION
**Pull request by IfhaamZ**

### CODE REVIEW

File: src/main/java/classes/Food.java
Comment (imalisina) [inline src/main/java/classes/Food.java:1]: remove the commented parts and send the completed code

File: src/main/java/classes/Foods.java
Comment (imalisina) [inline src/main/java/classes/Foods.java:1]: seed all of the available foods, then send the code

File: src/main/java/classes/Item.java
Comment (imalisina) [inline src/main/java/classes/Item.java:1]: Item class should be abstract

**Comment by imalisina** [comment]
Make changes and then send pull request

File: src/main/java/classes/Food.java
Comment (IfhaamZ) [reply src/main/java/classes/Food.java:1]: There are no commented parts

File: src/main/java/classes/Item.java
Comment (IfhaamZ) [reply src/main/java/classes/Item.java:1]: There are no abstract methods in Item

**Pull request closed without merging**